plugins/modules/mysql_info.py

```diff
@@ -138,6 +139,16 @@
       returned: if not excluded by filter
       type: int
       sample: 60
+    suffix:
+      description: Server suffix, e.g. MySQL, MariaDB, other or none.
```

Comment:
```suggestion
      description: Server suffix, for example, MySQL, MariaDB, other or none.
```
It's from Ansible best practices guide